server/src/main/java/org/apache/druid/metadata/IndexerSQLMetadataStorageCoordinator.java

```diff
@@ -1290,9 +1294,12 @@ public Void inTransaction(Handle handle, TransactionStatus transactionStatus)
 
   private void deleteSegment(final Handle handle, final DataSegment segment)
   {
-    handle.createStatement(StringUtils.format("DELETE from %s WHERE id = :id", dbTables.getSegmentsTable()))
-          .bind("id", segment.getId().toString())
-          .execute();
+    String segmentsTable = dbTables.getSegmentsTable();
+    String segmentId = segment.getId().toString();
+    log.debug("Removing segment [%s] [%s] from Metadata Storage [%s]!", segmentId, segment.getDataSource(), segmentsTable);
```

Comment:
nit: I guess this could be a lot of segments, it might be nicer to pull this outside of the loop and use `log.debugSegments`

Reply:
Nice idea, I didn't notice there is `log.debugSegments` in druid before. Changed.